SU2_CFD/src/solvers/CIncNSSolver.cpp

```diff
@@ -97,6 +98,43 @@ void CIncNSSolver::Preprocessing(CGeometry *geometry, CSolver **solver_container
 
   ComputeVorticityAndStrainMag<1>(*config, iMesh);
 
+  /*--- Compute recovered pressure and temperature for streamwise periodic flow ---*/
+  if (config->GetKind_Streamwise_Periodic() != NONE) {
+
```

Comment:
Move this block to a private function.
We should have code that looks like a nice tree, big classes use medium that use small, small functions use smaller functions.

Reply:
Yep made it a private function 
```c++
if (config->GetKind_Streamwise_Periodic() != NONE)
    Compute_Streamwise_Periodic_Recovered_Values(config, geometry, iMesh);
```